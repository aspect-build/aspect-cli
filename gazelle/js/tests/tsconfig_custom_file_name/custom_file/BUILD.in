# gazelle:js_tsconfig enabled
# gazelle:js_tsconfig_file bazel.tsconfig.json