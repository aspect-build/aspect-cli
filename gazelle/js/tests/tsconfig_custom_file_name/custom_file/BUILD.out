load("@aspect_rules_ts//ts:defs.bzl", "ts_config", "ts_project")

# gazelle:js_tsconfig enabled
# gazelle:js_tsconfig_file bazel.tsconfig.json

ts_project(
    name = "custom_file",
    srcs = ["index.ts"],
    tsconfig = ":bazel_tsconfig",
    deps = ["//:node_modules/tslib"],
)

ts_config(
    name = "bazel_tsconfig",
    src = "bazel.tsconfig.json",
)
